.github/workflows/modoboa.yml

```diff
@@ -97,7 +97,6 @@ jobs:
           pip install -r mysql-requirements.txt
           echo "DB=mysql" >> $GITHUB_ENV
       - name: Test with pytest
-        if: ${{ matrix.python-version != '3.8' || matrix.database != 'postgres' }}
```

Comment:
Why do you remove this line?

Reply:
If you look closely, you'll see that there is one job to run only tests, and one job to run tests and coverage